interface/patient_file/encounter/coding.php

```diff
@@ -1,6 +1,7 @@
 <?php
 
 /**
+ * TODO: @adunsulag need to find out if this file is used anywhere anymore since it links out to prescriptions and will be impacted by encounter navigation changes.
```

Comment:
I am not able to find this file being used anywhere, so can remove it.

Reply:
Wait, do not remove it. my phpstorm search just took awhile to find the places. It is used in encounter_bottom.php. This is called in 'Charges" menu item, which can turn on via the 'Use Charges Panel' global.